modules/setup/library/Setup/WebWizard.php

```diff
@@ -649,9 +636,10 @@ public function getRequirements($skipModules = false)
             )
         )));
 
-        $mysqlSet = new RequirementSet(true);
+        $isOptional = Platform::extensionLoaded('pdo_pgsql');
```

Comment:
Don't use `Platform` here. Create a new `RequirementSet` instead and set it as mandatory.

i.e.
```
$set ----- $dbSet
         /   OR  \
        /         \
$mysqlSet          $pgsqlSet
```